store-example04-textual-querying/start-client.sh

```diff
@@ -38,7 +38,7 @@ JAVAC="${JAVA_HOME}/bin/javac"
 
 uname | grep CYGWIN > /dev/null && TC_HOME=$(cygpath -w -p "${TC_HOME}")
 
-TC_CP=.
+TC_CP="$WD/src/"
 # Add the client jars in the classpath
 find "${TC_HOME}/client/" -type f -name "*.jar" > file
```

Comment:
`file` comment in `ehcache-example01-clustered-cache/start-client.sh` still applies ... needs some tweaking though since there are two sources of JAR files.  The `<( find "${TC_HOME}/client" -type f -name '*.jar' )` becomes `<( find "${TC_HOME}/client" -type f -name '*.jar' ; find "${TC_TCTQL_HOME}/client" -type f -name '*.jar' )`.

This, and the current method, yield duplicates in the resulting classpath since the TCTQL distro contains the TCStore client libs and SLF4J in addition to the TCTQL libraries -- that is, unless you've had it changed.